src/s2wasm.h

```diff
@@ -1342,6 +1346,40 @@ class S2WasmBuilder {
     }
   }
 
+  // Converts invoke wrapper names generated by LLVM backend to real invoke
```

Comment:
if this is wasm-backend specific code, perhaps it should be refactored out, in that `.s` files might in theory be generated by some other source. But I guess that other source would do the same thing. So maybe the comment can just be reworded to be more general?